### PR TITLE
Adds new Weaver items, makes the Ascent Gyne capable of weaving things.

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
As title says.

New obtainable objects include:
- Weaved Skinsuit
- Weaved Dress
- Silk Bindings
- Weaved Bandages

Skinsuit and Dress act primarily as flavor clothing. Bindings are weak (tape-strength) handcuffs that are not reusable, and bandages are a weaker/less numerous variant of basic bruise packs. I was originally going to include a pair of shoes and gloves as well but color stuff made them runtime for some reason; the sprites will remain for possible future use.

Additionally, Ascent Gynes are now weavers and can weave items; their weaver color is the same as razorweb and cannot be changed.
## Why It's Good For The Game
More weaver content is good.
## Did You Test It?
Yes.
## Authorship
ShadeAware
## Changelog

:cl:
/:cl:

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
Good to merge?

File: modular_mithra/code/modules/mob/living/carbon/human/species/trait_stuff/weaver_abilities.dm [hunk 1]
```diff
@@ -101,25 +101,53 @@
...
+			desired_result = /obj/item/clothing/under/weaverskinsuit
+			silkcost = 100
+			weavetime = 10
+			finalized = input(src, "Are you sure you want to weave a skinsuit? It will cost you [silkcost] silk.","Confirmation") as anything in list("Yes","No")
+		if(choice == "Dress")
+			desired_result = /obj/item/clothing/under/skirt_c/dress/long/gown/weaver
+			silkcost = 100
+			weavetime = 10
+			finalized = input(src, "Are you sure you want to weave a dress? It will cost you [silkcost] silk.","Confirmation") as anything in list("Yes","No")
+		/*if(choice == "Gloves")
```
Comment (Anton-Kr) [inline modular_mithra/code/modules/mob/living/carbon/human/species/trait_stuff/weaver_abilities.dm]: Either actually add these or remove the comments.

File: modular_mithra/code/modules/mob/living/carbon/human/species/trait_stuff/weaver_constructions.dm [hunk 2]
```diff
@@ -17,3 +17,27 @@
...
+
+/datum/weaver_recipe/item/skinsuit
+	title = "skinsuit"
+	result_type = /obj/item/clothing/under/weaverskinsuit
+
+/datum/weaver_recipe/item/dress
+	title = "dress"
+	result_type = /obj/item/clothing/under/skirt_c/dress/long/gown/weaver
+
+/*/datum/weaver_recipe/item/shoes
```
Comment (Anton-Kr) [inline modular_mithra/code/modules/mob/living/carbon/human/species/trait_stuff/weaver_constructions.dm]: ditto

File: code/modules/species/mantid/mantid.dm [hunk 3]
```diff
@@ -144,6 +144,7 @@
 	max_age =               500
 	blood_volume =         1200
 	spawns_with_stack =       0
+	is_weaver = 1
```
Comment (Anton-Kr) [inline code/modules/species/mantid/mantid.dm:147]: Use TRUE

Comment (ShadeAware) [reply code/modules/species/mantid/mantid.dm:147]: TRUE does not work in this instance. I don't know why.

Comment (Anton-Kr) [reply code/modules/species/mantid/mantid.dm:147]: I love byond

**Review by Anton-Kr — APPROVED** [review]

**Pull request merged**